src/ml_board/frontend/Dashboard/src/redux/status/statusSlice.ts

```diff
@@ -57,20 +54,20 @@ export const statusSlice = createSlice({
     upsertMetricOrLoss: (state, { payload }: PayloadAction<string[]>) => {
       state.metric_loss.push(...payload);
     }
-  }, extraReducers(builder) {
-    builder.addCase(upsertCharts, (state, { payload }) => {
-      // NOTE: very important to notice here that +4 increment
-      // because in the testing file every evaluation_result held 4 values for the same experiment
-      // this might not be true with other data
-      // TODO:
-      for (let i = 0; i < payload.length; i+=4) {
-        if (!state.color_map.hasOwnProperty(payload[i].exp_id)) {
-          // one can only hope that it doesn't produce a blue blue blue blue blue... pattern :')
-          state.color_map[payload[i].exp_id] = `#${Math.floor(Math.random() * 16777215).toString(16)}`;
-        }
-      }
-    })
-  },
+  }, 
+  // extraReducers(builder) {
+  //   builder.addCase(upsertCharts, (state, { payload }) => {
+  //     // NOTE: very important to notice here that +4 increment
+  //     // because in the testing file every evaluation_result held 4 values for the same experiment
+  //     // this might not be true with other data
+  //     for (let i = 0; i < payload.length; i+=4) {
+  //       if (!state.color_map.hasOwnProperty(payload[i].exp_id)) {
+  //         // one can only hope that it doesn't produce a blue blue blue blue blue... pattern :')
+  //         state.color_map[payload[i].exp_id] = `#${Math.floor(Math.random() * 16777215).toString(16)}`;
+  //       }
+  //     }
+  //   })
+  // },
```

Comment:
code that is commented out is legacy ;-)

Reply:
actually this bit of code was very hard to write 🥲 (namely lines 58, 59)
I wanted to leave them there just in case we needed to use them again?!